src/OpenTelemetry.Instrumentation.Http/MeterProviderBuilderExtensions.cs

```diff
@@ -63,11 +65,14 @@ public static class MeterProviderBuilderExtensions
                 HttpWebRequestActivitySource.MetricsOptions = options;
             });
         }
+#elif NET8_0_OR_GREATER
```

Comment:
Could we move this check to the top of the method and return the builder after adding the meters if it's on net8.0 or greater?